core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -90,6 +90,23 @@ static boolean hasIds(Schema schema) {
     return AvroCustomOrderSchemaVisitor.visit(schema, new HasIds());
   }
 
+  /**
+   * @param schema an Avro Schema
```

Comment:
Can you add a description like the other methods have?

Reply:
I tried to follow some example I saw under the [Catalog.class](https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/catalog/Catalog.java#L57), basically, a general description, then followed by `param`, `return`, `throws`. Let me know if I'm missing some components in this java doc.